package.json

```diff
@@ -2,28 +2,21 @@
   "name": "third-party-capital",
   "version": "1.0.26",
   "description": "",
-  "main": "./lib/cjs/index.js",
-  "types": "./lib/cjs/types/index.d.ts",
-  "files": [
-    "lib/**/*"
-  ],
+  "type": "commonjs",
   "exports": {
     ".": {
-      "import": {
-        "types": "./lib/esm/types/index.d.ts",
-        "default": "./lib/esm/index.js"
-      },
-      "require": {
-        "types": "./lib/cjs/types/index.d.ts",
-        "default": "./lib/cjs/index.js"
-      }
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs"
```

Comment:
you can provide a top level `types` option and a `types` here to typescript support, see https://github.com/nuxt/nuxt/blob/main/packages/nuxt/package.json#L20

Reply:
cool - adjusted the export to below:

```
"exports": {
    ".": {
      "import": "./dist/index.mjs",
      "require": "./dist/index.cjs",
      "types": "./dist/index.d.mts"
    }
  },
```